translations/help.yml

```diff
@@ -1,10 +1,10 @@
-description: 'Displays help for a command'
-help: "The <info>%command.name%</info> command displays help for a given command:\n\n<info>php %command.full_name% list</info>\n\nYou can also output the help in other formats by using the <comment>--format</comment> option:\n\n<info>php %command.full_name% --format=xml list</info>\n\nTo display the list of available commands, please use the <info>list</info> command.\n"
+description: 'आदेशासाठी मदत दर्शवितो.'
+help: "<info>% कमांड.नाव% </info> कमांड दिलेल्या कमांडसाठी मदत दर्शविते: \n \n <info> php% कमांड.पूर्णता @ सूची </info> \n \n आपण अन्य <comments> - स्वरूप </comments> पर्याय वापरून: \n \n <info> php% कमांड.संपूर्ण_नाव% --format = xml सूची </info> \n \n उपलब्ध आदेशांची सूची प्रदर्शित करण्यासाठी, कृपया <info> list </info> आदेश वापरा. ​​\n"
 arguments:
-    command_name: 'The command name'
+    command_name: 'आदेशाचे नाव'
 options:
-    xml: 'To output list as XML'
-    raw: 'To output raw command list'
-    format: 'The output format (txt, xml, json, or md)'
+    xml: 'क्सएमएलच्या रूपात आऊटपुट सूचीमध्ये.'
+    raw: 'कच्च्या आदेश सूचीचे आउटपुट करण्यासाठी.'
+    format: 'आउटपुट स्वरूप (txt, xml, json, किंवा md)'
 messages:
-    deprecated: 'The <comment>--xml</comment> option was deprecated in version 2.7 and will be removed in version 3.0. Use the <comment>--format</comment> option instead'
+    deprecated: '<comment> - xml </comment> पर्याय आवृत्ती 2.7 मध्ये नापसंत करण्यात आला आणि आवृत्ती 3.0 मधून काढले जाईल. त्याच्याऐवजी <comment> - स्वरूप </comment> पर्याय वापरा'
```

Comment:
<comment>-- xml</comment> must not be changed, same for <comment>--format</comment>